solr/core/src/java/org/apache/solr/core/ConfigSetService.java

```diff
@@ -344,8 +344,9 @@ protected NamedList loadConfigSetFlags(CoreDescriptor cd, SolrResourceLoader loa
   public abstract List<String> listConfigs() throws IOException;
 
   /**
-   * Get the names of the files in config including dir (mutable, non-null)
-   * e.g. solrconfig.xml, lang/stopwords_en.txt, lang/stoptags_en.txt
+   * Get the names of the files in config including dirs (mutable, non-null)
+   * Sorted lexicographically
+   * e.g. solrconfig.xml, lang/, lang/stoptags_en.txt, lang/stopwords_en.txt
```

Comment:
oh more more little thing.  I've never seen a file in Solr called "stoptags_en.txt" and just recently noticed there is one or two "stoptags_ja.txt" so that would better.  But it's extra; we only need _one_ file in the example in the subdir, so stopwords_en.txt is common & sufficient without "stoptags" (whatever that is).